src/main/java/com/wdtinc/mapbox_vector_tile/adapt/jts/UserDataKeyValueMapConverter.java

```diff
@@ -8,18 +8,22 @@
 import java.util.Objects;
 
 /**
- * Convert simple user data {@link Map} where the keys are {@link String} and values are {@link Object}. Supports
- * converting a specific map key to a user id. If the key to user id conversion fails, the error occurs silently
- * and the id is discarded.
+ * Convert simple user data {@link Map} where the keys are {@link String} and values are
+ * {@link Object}. Supports converting a specific map key to a user id. If the key to user id
+ * conversion fails, the error occurs silently and the id is discarded.
  *
  * @see IUserDataConverter
  */
 public final class UserDataKeyValueMapConverter implements IUserDataConverter {
 
-    /** If true, set feature id from user data */
+    /**
```

Comment:
pretty sure the Javadoc lint checks failed the build re multiline comment with single line